main.go

```diff
@@ -10,10 +11,13 @@ const PORT = ":8080"
 func main() {
 	// Initialization steps:
 	// - [x] Register routes in the http.ServeMux called "router"
-	// - [ ] Connect to database
+	// - [x] Connect to database
 	// - [x] Listen for connections on PORT
 	router := initRouter()
 
-	fmt.Printf("Server started on PORT %v...\n", PORT)
+	db := database.NewDatabase()
+	defer db.Close()
+
+	log.Printf("Server started on PORT %v...\n", PORT)
 	http.ListenAndServe(PORT, router)
```

Comment:
```
log.Fatal(http.ListenAndServe(port, router)
```

Constants aren't all caps in Go (I know, I hate it too).

`ListenAndServe` blocks, so if it returns something went wrong. We wrap it in `log.Fatal` so that our log prints out what happened, followed by termination.